unicodetools/data/emoji/13.1/emoji-zwj-sequences.txt

```diff
@@ -1,11 +1,11 @@
 # emoji-zwj-sequences.txt
```

Comment:
The processing step is to take the generated files from the emoji staging repro and copy it into the unicodetools internal directory. This is done *before* flipping the switch to exit beta.